fundingmanager.go

```diff
@@ -2025,7 +2031,12 @@ func (f *fundingManager) handleFundingSigned(fmsg *fundingSignedMsg) {
 		fndgLog.Infof("Broadcasting funding tx for ChannelPoint(%v): %x",
 			completeChan.FundingOutpoint, fundingTxBuf.Bytes())
 
-		err = f.cfg.PublishTransaction(fundingTx, "")
+		label := labels.MakeLabel(
+			labels.LabelTypeChannelOpen,
+			completeChan.ShortChanID().ToUint64(),
```

Comment:
The short channel ID is always 0 here, because it's not confirmed yet. I think we should update the label (they can be updated, right?) in `handleFundingConfirmation`, right after `f.cfg.ReportShortChanID(fundingPoint)`.
We should have access to the funding TX there and know what label to overwrite.

Reply:
We can overwrite, thanks for pointing this out! Added an update after we have our short chan ID.